docs/fides/docs/quickstart/docker.md

```diff
@@ -26,6 +26,16 @@ Initializing Fidesctl...
 
 Created a '.fides' directory.
 
+Fides needs your permission to send Ethyca a limited set of anonymous usage statistics.
```

Comment:
would it be beneficial to reiterate the privacy consciousness of the usages statistics? something like:

`Fides needs your permission to send Ethyca a limited set of privacy-conscious, anonymous usage statistics.`

Reply:
I'm not sure this really changes the meaning of the sentence, does it? I think by including a link to the repo, we invite users to dig into the privacy respecting nature of the service.

Reply:
No, it doesn't change the meaning. I thought would go well with the product marketing as well as repeating the phrase in several locations could help drive home the philosophy and encourage people to not opt out.  But that's pure speculation.

Reply:
I'm hesitant to change this because it was dictated to me. The nice thing is that it's maintained in one place (in fideslog), so it would be easy to make a change if the higher-ups decide to change it.